BunnyHopMaster/Assets/Scripts/Player/PlayerConstants.cs

```diff
@@ -62,11 +63,21 @@ public static class PlayerConstants
     public static string MouseX = "Mouse X";
     public static string MouseY = "Mouse Y";
 
+    public static string WinMenu = "WinMenu";
+    public static string WinMenuDefault = "O";
+
+    public static string NextLevel = "NextLevel";
+    public static string NextLevelDefault = "L";

```

Comment:
updated the ticket to provide easier to use hotkeys: https://github.com/ambid17/BunnyHopMaster/issues/129

Added another ticket to update the UI to let them know these hot keys exist: https://github.com/ambid17/BunnyHopMaster/issues/134